.changeset/pre.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mode": "pre",
+  "tag": "rc",
+  "initialVersions": {
+    "@shopify/ui-extensions": "2025.10.0-rc.0"
+  },
+  "changesets": []
```

Comment:
What's this for?

Reply:
[this was generated by `changeset pre enter next` ](https://changesets-docs.vercel.app/en/prereleases#:~:text=bun-,npm%20run%20changeset%20pre%20enter%20next,-npm%20run%20changeset)

It gets populated when `changeset version` is run. I imagine its to keep track of what needs to be released? I'm not entirely sure tbh.